FoliCon/Modules/utils/DataUtils.cs

```diff
@@ -1,9 +1,47 @@
-﻿namespace FoliCon.Modules.utils;
+﻿using NLog.Fluent;
+using System.Windows.Media;
+using static Vanara.PInvoke.User32;
+
+namespace FoliCon.Modules.utils;
 
 public static class DataUtils
 {
     public static int GetResultCount(bool isPickedById, dynamic result, string searchMode)
     {
         return isPickedById ? result != null ? 1 : 0 : searchMode == "Game" ? result.Length : result.TotalResults;
     }
+
+    /// <summary>
+    /// Internal helper to format the Rating string for the selected Item.
+    /// The decimal points of rating should be limited to 2 digits : https://github.com/DineshSolanki/FoliCon/issues/145
+    /// Eg: X.XX -> 7.15
+    /// </summary>
+    /// <param name="ratingInput">Rating value.</param>
+    /// <returns>Formatted Rating value.</returns>
+    public static string FormatRatingString(string ratingInput)
```

Comment:
try to find a simpler solution to the same problem, checking for digits etc, 
TIP - even though the type of rating is string, it will always be a number.